### PR TITLE
*: clean up doc.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut  @queenypingcap @shenli

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
 
 When a node starts, the metadata of node, store and region are registered into PD. The status of each Region and Store is reported to PD regularly.
```
Comment (queenypingcap) [inline README.md]: When a node starts, the metadata of the Node, Store and Region are registered into PD. The status of each Region and Store is reported to PD regularly.

File: README.md [hunk 2]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
...
 
-### Usage
+### Build
 
-See [Usage](./USAGE.md) for how to install and run TiKV.
+TiKV is a component in TiDB project, you must build and run it with TiDB and PD together.
```
Comment (queenypingcap) [inline README.md]: TiDB project -> the TiDB project

File: README.md [hunk 3]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
...
+
+The easily way to build TiDB project is to follow [user build guide](https://github.com/pingcap/docs/blob/master/op-guide/build.md).
```
Comment (queenypingcap) [inline README.md]: +If you want to use TiDB in your project, see [user build guide](https://github.com/pingcap/docs/blob/master/op-guide/build.md) to build the TiDB project first.

File: README.md [hunk 4]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
...
+If you are a developer, you can also follow [dev build guide](https://github.com/pingcap/docs/blob/master/dev-guide/build.md).
```
Comment (queenypingcap) [inline README.md]: +If you want to dive into TiDB, see [dev build guide](https://github.com/pingcap/docs/blob/master/dev-guide/build.md) on how to build the TiDB project.

File: README.md [hunk 5]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
...
+
+### Next steps
+
++ Read the [clustering doc](https://github.com/pingcap/docs/blob/master/op-guide/clustering.md) on how to run TiDB project.
```
Comment (queenypingcap) [inline README.md]: TiDB project -> the TiDB project

File: README.md [hunk 6]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
...
++ Learn the [configuration explanation](https://github.com/pingcap/docs/blob/master/op-guide/configuration.md).
```
Comment (queenypingcap) [inline README.md]: configuration explanation ->configuration explanations

File: README.md [hunk 7]
```diff
@@ -37,9 +37,18 @@ This figure represents tikv-server software stack.
...
++ Use [docker](https://github.com/pingcap/docs/blob/master/op-guide/docker.md) to run TiDB project.
```
Comment (queenypingcap) [inline README.md]: docker ->Docker

**Comment by ngaut** [comment]
LGTM

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**